packages/perseus/src/widgets/__tests__/__snapshots__/matrix_test.jsx.snap

```diff
@@ -94,7 +94,7 @@ exports[`matrix widget should snapshot on mobile: first mobile render 1`] = `
                       autocapitalize="off"
                       autocomplete="off"
                       autocorrect="off"
-                      class="input_1t4b01j-o_O-LabelMedium_1rew30o-o_O-default_1r2dwp3-o_O-input_bqz18q"
+                      class="input_1t4b01j-o_O-LabelMedium_1rew30o-o_O-default_1r2dwp3-o_O-input_9b0qf5"
```

Comment:
**thought**: This is getting me thinking.  It'd almost be handy if our snapshots included the CSS class(es) somewhere so that we could compare those instead of just seeing that a generated classname has changed. 

No action required.